### PR TITLE
[bitnami/kubectl] Add VIB tests

### PR DESCRIPTION
**Pull request by andresbono**
### Description of the change

The main objective of this PR is to publish our Bitnami Kubectl container using VMware Image Builder. In order to do that, several changes are included:

- Increasing the existing test coverage of the asset by adding Goss tests.
- Update verify and publish VIB pipeline's definitions.

### Benefits

- Ensuring higher quality of the container catalog.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

NA

### CODE REVIEW

File: .vib/kubectl/goss/vars.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+binaries:
+  - kubectl
+  - jq
```
Comment (andresbono) [inline .vib/kubectl/goss/vars.yaml:3]: jq is installed via system packages, not sure if it is worth checking it here. WDYT?

Comment (FraPazGal) [reply .vib/kubectl/goss/vars.yaml:3]: We don't usually check system pkgs installed via Dockerfile with the exception of ca-certificates, which enables https. The reason for this is that testing Dockerfile logic means we would be checking one of the following:

1. Base image distro
2. Docker logic

Both of them are external to the asset itself and are expected to work as expected and thus should not require explicit testing in our side.

That said, this is the current approach but please share your thoughts on this.

File: .vib/kubectl/goss/vars.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+directories:
+  - mode: "0775"
+    paths:
+      - /.kube
```
Comment (FraPazGal) [inline .vib/kubectl/goss/vars.yaml:7]: I couldn't find our logic modifying this directory, could you point me to it?

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @andresbono! I added a couple of comments as review, please check them when you can.

File: .vib/kubectl/goss/vars.yaml (hunk 1 above)
Comment (andresbono) [reply .vib/kubectl/goss/vars.yaml:3]: Thank you! I didn't have strong thoughts on this. The current approach makes sense to me 👍

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**